cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseTableAdmin.java

```diff
@@ -81,6 +81,7 @@ public void create() throws IOException {
 
 
     final HTableDescriptor tableDescriptor = new HTableDescriptor(name);
+    setVersion(tableDescriptor);
```

Comment:
yes, we were not adding cdap version property to tables up until now.. we only did on table upgrade